src/Inputs/Slider.tsx

```diff
@@ -1,79 +1,379 @@
+                                left={-11}
+                                top={-50}
+                                height={20}
+                                width={50}
+                                {...props}
+                            >
+                                {leftKnobPosition}
+                            </Popup>
+                        )}
+                    </SliderKnobLeft>
+                )}
+
+                <SliderKnobRight
+                    disabled={disabled}
+                    hasTwoKnobs={hasTwoKnobs}
+                    left={translateToPixels(rightKnobPosition)}
+                    onMouseDown={(event): void => handleMouseDown(event)}
+                >
+                    {hasPopup && (
+                        <Popup
+                            left={-11}
+                            top={-50}
+                            height={20}
+                            width={50}
+                            {...props}
+                        >
+                            {rightKnobPosition}
+                        </Popup>
+                    )}
+                </SliderKnobRight>
+            </SliderBoard>
+            {marks && (
+                <SliderBoardMarks
+                    ref={marksBarRef}
+                    theme={theme}
+                    disabled={disabled}
+                >
+                    {marks.map(
+                        ({ key, mark }): React.ReactElement => (
+                            <div
+                                key={key}
+                                style={{
+                                    position: 'absolute',
+                                    left: `${translateToPixels(key)}px`,
+                                }}
+                            >
+                                {mark}
+                            </div>
+                        ),
+                    )}
+                </SliderBoardMarks>
+            )}
+        </LabelLayout>
+    );
+};
+
+const SliderBoard = styled.div<SliderProps>`
         width: 100%;
-        height: 8.4px;
-        cursor: pointer;
+        height: 4px;
         animate: 0.2s;
-        box-shadow: 1px 1px 1px #000000, 0px 0px 1px #0d0d0d;
-        background: #3071a9;
-        border-radius: 1.3px;
-        border: 0.2px solid #010101;
+        background: #e9e9e9;
+        cursor:pointer;
+        ${({ theme }): string => `
+            border-radius: ${theme.dimensions.radius};
+            box-shadow: ${theme.depth[1]};
+        `}
+        // Disabled
+        ${({ disabled }): string =>
+            disabled
+                ? `
+            cursor: not-allowed;
+            opacity: 0.6;
+        `
+                : ''}
+        `;
+
+export interface SelectedBarProps {
+    left: number;
+    right: number;
+    disabled: boolean;
+    hasRail: boolean;
+}
+
+const SliderBoardSelected = styled.div<SelectedBarProps>`
+    top: 1px;
+    height: 4px;
+    animate: 0.2s;
```

Comment:
here as well, I opened up your branch and it's giving me unknown property, can you just check to make sure this is valid?
![image](https://user-images.githubusercontent.com/47584841/83686094-a1c71b00-a5b7-11ea-8d66-877ca178aa7a.png)